### PR TITLE
Completed fetch-in-react.

### PR DESCRIPTION
**Pull request by leanne-vu**
Screen Shot 2022-12-19 at 2 48 50 PM

### CODE REVIEW

File: fetch-in-react/src/app.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      users: [],
+      isLoading: true
+    };
+  }
+
+  componentDidMount() {
+
+    fetch('https://jsonplaceholder.typicode.com/users')
+      .then(response => response.json())
+      .then(data => this.setState({ isLoading: !this.state.isLoading, users: data }))
```
Comment (Zhiwutian) [inline fetch-in-react/src/app.jsx:17]: you don't want to flip here, it should only set to true, because that means the data is done loading.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small update!

**Comment by leanne-vu** [comment]
Screen Shot 2022-12-19 at 3 05 07 PM

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**